explorer/interpreter/impl_scope.cpp

```diff
@@ -11,7 +11,6 @@
 
 using llvm::cast;
 using llvm::dyn_cast;
```

Comment:
Maybe we should move these to a header file and pull them into namespace `Carbon`? It's awkward pulling just the ones that a particular file needs into that file, and adding and removing them as uses get added / removed.

Clang does something similar here: https://clang.llvm.org/doxygen/LLVM_8h_source.html